From 7587228433ad150c1a48696c3ff5c768739279e6 Mon Sep 17 00:00:00 2001
From: John Thacker <johnthacker@gmail.com>
Date: Thu, 16 May 2024 14:11:19 +0000
Subject: [PATCH] CMake: Look for the lua library with versioned suffixes

Alpine Linux puts not just the include files but also the
library in a versioned subdirectory, so look for it there too.

Fix #19841


(cherry picked from commit 59b376dce046eb4f739a70f49c20ac9da9cab794)

Co-authored-by: John Thacker <johnthacker@gmail.com>
---
 cmake/modules/FindLua.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindLua.cmake b/cmake/modules/FindLua.cmake
index 23e50a6a947..6a594a7d546 100644
--- a/cmake/modules/FindLua.cmake
+++ b/cmake/modules/FindLua.cmake
@@ -184,7 +184,7 @@ find_library(LUA_LIBRARY
   HINTS
     ${LUA_HINTS}
     ENV LUA_DIR
-  PATH_SUFFIXES lib
+  PATH_SUFFIXES lib ${_lua_library_names}
 )
 unset(_lua_library_names)
 
-- 
GitLab

